src/ai/backend/client/func/vfolder.py

```diff
@@ -177,7 +179,16 @@ async def download(
             })
             async with rqst.fetch() as resp:
                 download_info = await resp.json()
-                download_url = URL(download_info['url']).with_query({
+                overriden_url = download_info['url']
+                if address_map is not None:
```

Comment:
The unconfigured default value in `APIConfig.DEFAULTS` should match with the rhs of this conditional statement.